python/lsst/meas/algorithms/objectSizeStarSelector.py

```diff
@@ -37,6 +36,11 @@
 
 
 class ObjectSizeStarSelectorConfig(BaseSourceSelectorTask.ConfigClass):
+    doFluxLimit = pexConfig.Field(
```

Comment:
Maybe a better name for these variables would be, as you say in the comment, applyFluxLimit and applySignalToNoiseLimit. It makes it slightly clearer what they are for where "do" seems to apply a sort of processing.

Reply:
Just to be thorough, @PaulPrice, do you have any strong feelings about the above?

Reply:
I don't think it's worth the trouble of changing; surely we have much more important things to do.

Reply:
So you would leave the **do**s?

Reply:
@morriscb are you ok with me leaving the ticket essentially as is?

Reply:
Is there a good reason to change the names? Good enough to spend a few days on an RFC and implementation and backwards compatibility and support?

Reply:
I was fine following the convention you used and sticking to it would cause less churn.  The RFC is more about adapting the selectors to make (homogeneous) use of the base and derived tasks you defined in **sourceSelector.py** (API changing and thus RFC-required).  I also want to change the default selection criterion to be S/N instead of flux for this particular selector, but I'll probably need to run at least the HSC RC2 dataset to make sure things still look good after this change.

I'll wait to see if Chris is ok with keeping the status quo...

Reply:
I feel that the extra booleans make the config fairly cluttered and are also not very descriptive as to what they are doing. I would prefer the option below instead where if the value is set and not `None`, the cuts are applied vs having the an extra boolean that needs to be set in addition to the values. The extra booleans could lead to silent failures where the one expects the selection to be applied but it is not because the boolean was not set. The likelihood of this happening is increased by one selection being default `True` and the other being default `False`. I'm not sure how necessary an RFC is for this but feel free to ask around. The option below of testing `None` would seem to work in cases where the task this selector is being called by sets it's own default for the value.

Reply:
Ok, I think I can get on board with getting rid of the booleans here an having the None/not None-ness of the limits determine whether a cut is made (thus making the naming discussion moot).

Reply:
Allrighty...have a look and see if you're ok with it now.  Jenkins is running:
https://ci.lsst.codes/blue/organizations/jenkins/stack-os-matrix/detail/stack-os-matrix/29231

Reply:
I really don't like the idea of having `None`-ness disable a selection. It is extremely convenient to be able to disable a selection on the command-line through a boolean flag. Also, having a boolean flag allows you to disable a selection while preserving its parameters.

Reply:
Okay, it's not quite directly your case Paul, but in general I can see how using `None` to disable the cut could be fairly awkward and require a bit more in depth parsing of the config rather than just glancing at the config and setting the boolean. The tickets is already marked as `Reviewed` so I'm happy to use Laruen's reasoning above that the other selectors use these variables to move along.